std/algorithm/searching.d

```diff
@@ -3969,6 +3984,15 @@ if (is(typeof(binaryFun!pred(r1.front, r2.front))) &&
     assert(r1 == ["abc", "def", "hij"]);
     assert(skipOver!((a, b) => a.equal(b))(r1, r2));
     assert(r1 == ["def", "hij"]);
+
+    import std.ascii : isWhite;
+
+    auto s2 = "\t\tvalue";
+    auto s3 = "";
+    auto s4 = "\t\t\t";
+    assert(s2.skipOver!isWhite && s2 == "value");
+    assert(!s3.skipOver!isWhite);
+    assert(s4.skipOver!isWhite && s3.empty);
```

Comment:
You need to import empty from std.range.primitives here (public tests need to be runnable standalone)